ui/hooks/src/use-comments.new.ts

```diff
@@ -98,14 +84,25 @@ export interface Publish_Options {
   data: DataProviderInput[];
   comment: { title?: string; tags?: string[]; postID: string };
 }
-export function useDeleteComment(commentID: string) {
+
+const deleteComment = async (commentId: string) => {
   const sdk = getSDK();
+  const resp = await lastValueFrom(sdk.api.comments.removeComment(commentId));
+  if (resp.hasOwnProperty('error')) {
+    throw new Error(resp['error']);
```

Comment:
Where are these errors caught?

Reply:
https://react-query.tanstack.com/guides/query-functions#handling-and-throwing-errors